orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -456,7 +442,10 @@ def _write_function_body_return_multiple(self, writer, cpp_func, in_place_params
             writer.write(in_place_params[key])
         writer.writeline(");")
 
-    def _need_set_out_tensor(self, first_param, last_param, return_info):
+    # determines if the "out" param exists and needs to be set.
+    # "out", while similar to other modifiable (e.g. 'in_place') params, is treated
+    # differently because the intent is to put result data into it, not modify data
+    def _should_set_out_tensor(self, first_param, last_param, return_info):
         # if the torch func has a return ref tensor, out is the last param, and self param is the first input
         # then we need to update and return out. Record this need in set_out_tensor.
```

Comment:
nit: Delete `Record this need in set_out_tensor.` as this is no longe done in this method but by the caller of this method.